### PR TITLE
"What is a header?"

### PR DESCRIPTION
**Pull request by indiegamehunt**

### CODE REVIEW

File: what-header/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## What is a header?
```
Comment (MaskofThoro) [inline what-header/index.md]: this is called "Page Header" on wttf, should we update here to stay consistent?

**Comment by MaskofThoro** [comment]
can we update the title to Page Header? so it's help.gamejolt.com/pageheader

**Comment by indiegamehunt** [comment]
Comments addressed in edits. Combining with #22 and #23

**Comment by indiegamehunt** [comment]
Ready to review

File: src/page-header/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## What are the header requirements and guidelines?
+
+Here are the requirements and recommendations for header images:
+
+- The image format can be PNG, JPG, or GIF. PNGs will result in the highest quality. 
+- In general, your image should be as large as possible. 
+- The minimum recommended width is 1920px. Images less wide than this will be automatically enlarged. The maximum width is 5120px.
+- The minimum ratio (width/height) is 2.7 and the maximum ratio is 6.3. The recommended ratio is 4. 
+	- We recommend making the width 400% of the height (it has to be between 271.5% and 630%).
```
Comment (MaskofThoro) [inline src/page-header/index.md]: Can we swap this to something like "We recommend making the height 25% of the width" ? The three digit percentages are a bit scary to look at.

File: src/page-header/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Headers are the big, banner-like images that adorn the tops of game pages. They are optional but highly, highly recommended. Headers make for a splendid looking page.
+
+You might want to experiment with different images to see what looks good as a header. Screenshots often turn out well, but some of the nicest looking headers are created for the purpose. 
+
+## What are the header requirements and guidelines?
+
+Here are the requirements and recommendations for header images:
+
+- The image format can be PNG, JPG, or GIF. PNGs will result in the highest quality. 
+- In general, your image should be as large as possible.
```
Comment (MaskofThoro) [inline src/page-header/index.md]: maybe mention that they need to be rectangles?

File: src/page-header/index.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- The minimum ratio (width/height) is 2.7 and the maximum ratio is 6.3. The recommended ratio is 4. 
+	- We recommend making the width 400% of the height (it has to be between 271.5% and 630%). 
+    - To find a good height, divide the width by 4 (it can actually be any number between 2.7 and 6.3).
+    - If you have the height and need a good width, just multiply the height by 4 (or anything between 2.7 and 6.3).
+- Make sure anything important is located near the center of the image. This will ensure your header looks its best on mobile devices.
+- Only use an original image created or captured by you or your team. 
+
+## Making a quick header
+
+If you don't want to make a custom image for your header, use a screenshot. A lot of screenshots turn out really nicely when blown up to header size.
```
Comment (MaskofThoro) [inline src/page-header/index.md]: can't blow up a screenshot to a header size because it will be blurry.

File: src/page-header/index.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# What is a page header?
+
+Headers are the big, banner-like images that adorn the tops of game pages. They are optional but highly, highly recommended. Headers make for a splendid looking page.
```
Comment (MaskofThoro) [inline src/page-header/index.md]: Mention they are rectangles "Headers are the big, banner-like rectangle images that adorn the tops of game pages."

**Pull request merged**